05-JWT-Basics/starter/errors/bad-request.js

```diff
@@ -0,0 +1,11 @@
+const CustomAPIError = require("./custom-error")
+const { StatusCodes } = require("http-status-codes")
+
+class BadRequestError extends CustomAPIError {
+  constructor(message) {
+    super(message)
+    this.statusCode = StatusCodes.BAD_REQUEST
```

Comment:
Always try to keep in mind what the actual status codes are, since it's very common to refer to them by number

- 200; ok
- 201; created
- 400; bad request
- 403; forbidden
- 404; not found
- 500; server error

Those are the most common / important to know.

Reply:
These status codes are part of the HTTP standard, so they are used across the web and across programming languages / technologies!